docs/contribute/documentation/new-snippets.mdx

```diff
@@ -84,7 +84,7 @@ Most code examples should include a file name so readers can understand which fi
 
 ### Language configuration
 
-Use the `language` attribute to define the language to which the code snippet applies. The documentation uses this attribute to determine which variant to display (e.g., JavaScript, TypeScript, TypeScript 4.9, MDX).
+Use the `language` attribute to define the language to which the code snippet applies. The documentation uses this attribute to determine which variant to display (e.g., JavaScript, TypeScript, MDX).
```

Comment:
@valentinpalkovic just a minor thing, can you remove the `ts-4-9` reference from the example below so that it aligns with this section? Turning it into:
````md title="docs/_snippets/button-stories.md"
```ts filename="Button.stories.ts" language="js|ts|mdx"
```
````

Reply:
Sure, I will do so :)